linkerd/docs/config.md

```diff
@@ -662,23 +662,20 @@ experimental.
 
 The Kubernetes namer is configured with kind `io.l5d.experimental.k8s`, and these parameters:
 
-* *host* -- the Kubernetes master host. (default: kubernetes.default.svc.cluster.local)
-* *port* -- the Kubernetes master port. (default: 443)
-* *tls* -- Whether TLS should be used in communicating with the Kubernetes master. (default: true)
-* *tlsWithoutValidation* -- Whether certificate-checking should be disabled. (default: false)
-* *authTokenFile* -- Path to a file containing the Kubernetes master's authorization token.
-  (default: /var/run/secrets/kubernetes.io/serviceaccount/token)
+* *host* -- the Kubernetes master host. (default: localhost)
+* *port* -- the Kubernetes master port. (default: 8001)
 
 For example:
 ```yaml
 namers:
 - kind: io.l5d.experimental.k8s
   host: kubernetes.default.svc.cluster.local
   port: 443
```

Comment:
I think it would be worthwhile to change this to `host: localhost` `port: 8001`, since 443 is typically SSL and we stopped supporting that.